modules/gha/main.go

```diff
@@ -674,6 +683,16 @@ func (p *Pipeline) warmEngineStep() JobStep {
 	return p.bashStep("warm-engine", nil)
 }
 
+func (p *Pipeline) dockerhubAuthStep() []JobStep {
+	return []JobStep{
+		p.bashStep("dockerhub-auth", map[string]string{
+			"DOCKERHUB_USERNAME": "${{ vars.DOCKERHUB_USERNAME }}",
+			"DOCKERHUB_PASSWORD": "${{ secrets.DOCKERHUB_PASSWORD }}",
+			"DOCKERHUB_SERVER":   "${{ vars.DOCKERHUB_SERVER }}",
+		}),
+	}
+}
```

Comment:
Could we somehow allow the user to specify these?

We shouldn't assume the secrets layout from the gha module, since it's meant to be a re-usable module.

Reply:
Rather than doing that, I would much rather remove this entirely and add a guard if the file does not exist.